Documentation/cmdref/cilium-agent.md

```diff
@@ -113,6 +113,7 @@ cilium-agent [flags]
       --k8s-watcher-queue-size uint                           Queue size used to serialize each k8s event type (default 1024)
       --keep-bpf-templates                                    Do not restore BPF template files from binary
       --keep-config                                           When restoring state, keeps containers' configuration in place
+      --kube-proxy-replacement string                         Detect and enable individual features for kube-proxy replacement ("strict"), or enable only selected features and panic if dependencies between features are broken ("strict"), or enable only selected features regardless dependencies ("partial")  (default "strict")
```

Comment:
```suggestion
      --kube-proxy-replacement string                         Detect and enable individual features for kube-proxy replacement ("probe"), or enable only selected features and panic if dependencies between features are broken ("strict"), or enable only selected features regardless dependencies ("partial")  (default "strict")
```

Reply:
Is there any reason to do "partial"? What does that enable users to do? They'll still need kube-proxy.

Reply:
The main idea of "partial" is to run together with kube-proxy and to offload only some its functionality to cilium.

E.g. a user wants to enable NodePort for outside connections (handled by bpf_netdev), but at the same time they want to disable the host-reachable services (e.g. due to https://github.com/cilium/cilium/issues/9974) and instead to use kube-proxy for accesing NodePort from inside cluster.